en/sim_gazebo_gz/worlds.md

```diff
@@ -0,0 +1,80 @@
+# Gazebo Worlds
+
+This topic provides imagery/information about the [Gazebo](../sim_gazebo_gz/index.md) worlds supported by PX4.
+
+The [default world](#default) is spawned by default, though this may be overridden by a [model specific world](#model_specific_worlds).
+Developers can also manually specify the world to load: [Gazebo > Specify World](../sim_gazebo_gz/index.md#specify-world) (or [Gazebo Models Repository](../sim_gazebo_gz/gazebo_models.md#gazebo-models-repository-px4-gazebo-models)).
+
+The source code for supported worlds can be found in the [Gazebo Models Repository](../sim_gazebo_gz/gazebo_models.md#gazebo-models-repository-px4-gazebo-models) on GitHub here: [PX4/PX4-gazebo-models/tree/main/worlds](https://github.com/PX4/PX4-gazebo-models/tree/main/worlds).
+
+## Empty (Default) {#default}
+
+Empty world (a grey plane).
+This is used by default.
+
+[PX4-gazebo-models/main/worlds/default.sdf](https://github.com/PX4/PX4-gazebo-models/blob/main/worlds/default.sdf)
+
+![screenshot of default world](../../assets/simulation/gazebo/worlds/default.png)
+
+## Aruco
+
+TBD: What's it for? What vehicles should it be used with. Is it used by any vehicles by default?
+
+[PX4-gazebo-models/main/worlds/aruco.sdf](https://github.com/PX4/PX4-gazebo-models/blob/main/worlds/aruco.sdf)
+
+![screenshot of Aruco world](../../assets/simulation/gazebo/worlds/aruco.png)
+
+## Baylands
+
+Baylands world surrounded by water.
+
+[PX4-gazebo-models/main/worlds/bayland.sdf](https://github.com/PX4/PX4-gazebo-models/blob/main/worlds/baylands.sdf)
+
+![screenshot of Baylands world](../../assets/simulation/gazebo/worlds/baylands.png)
+
+## Lawn
+
+TBD: What's it for? What vehicles should it be used with. Is it used by any vehicles by default?
+
+[PX4-gazebo-models/main/worlds/lawn.sdf](https://github.com/PX4/PX4-gazebo-models/blob/main/worlds/lawn.sdf)
+
+![screenshot of lawn world](../../assets/simulation/gazebo/worlds/lawn.png)
+
+## Rover
+
+TBD: What's it for? What vehicles should it be used with. Is it used by any vehicles by default?
```

Comment:
What's it for? What vehicles should it be used with. Is it used by any vehicles by default?

How does this differ from Lawn world? Which should you use, and when?

Reply:
The rover world is very similiar to the lawn world with these 2 important differences differences:
- Grid on the ground which is useful as a reference while driving.
- Higher update rate which solves segfault issues specifically with rovers with ackermann steering.

Currently the r1 rover (4009) and ackermann rover (4012) use it as default and all future ground vehicles I'll work with will have it as default aswell.
I will most likely add more rover friendly changes to it down the line since there are still some issues with rover simulation.

In summary: It is a world that is and will be further optimized for ground vehicles.

Reply:
Thanks. FYI I see a grid on lawn. I do see segfaults, but it worked for ackermann, not r1.